File: python/instruccion.py
from abc import ABCMeta, abstractmethod


class Instruccion:
    @abstractmethod
    def execute_inst(self) -> None:
        pass


class Left(Instruccion):
    @classmethod
    def execute_inst(robot):
        pass


class Right(Instruccion):
    @classmethod
    def execute_inst(robot):
        pass


class Forward(Instruccion):
    @classmethod
    def execute_inst(robot):
        pass

File: node/src/logica/Instruccion.js
class Instruccion {
    LEFT = "LEFT"
    RIGHT = "RIGHT"
    FORWARD = "FORWARD"

    static inst_method = {
        "LEFT": "move_left()",
        "RIGHT": "move_right()",
        "FORWARD": "move_forward()"
    }

    static prueba = 'mongoler'

    constructor(instruccion) {
        this.instruccion = instruccion
    }
}

module.exports = Instruccion